### ISSUE TITLE
Process approachsettlement-v1.0

### ISSUE THREAD

**Issue by pveeckhout** [eddb2-0/EDDB-2.0-backend#70]
https://github.com/EDCD/EDDN/blob/master/schemas/approachsettlement-v1.0.json

map the approachsettlement EDDN message to POJO conform to CommodityV3

**Comment by jakaarl** [comment]
Yeah, so turns out the POJO mapping is there, and since I already started working on the processor, I renamed this issue accordingly.
